### DISCUSSION TITLE
Static webpage example - link?

### DISCUSSION THREAD

**Discussion by mskyttner** [githubocto/flat-action#31] [Q&A] [answered]
At https://octo.github.com/projects/flat-data it says "If you have a static webapp, you can power it with data that is kept evergreen by Flat (we built a simple example using GitHub pages)"... 

Would it be possible to put the link to that example in there?

I looked around at https://github.com/githubocto?q=flat but couldn't find that example (yet).

**Comment by irealva** [comment, marked as answer]
Absolutely! This is our demo [Covid Dashboard](https://github.com/githubocto/flat-demo-covid-dashboard) repo. Let me know if it's what you were looking for. It uses Flat to injest data into the repo and then uses [GitHub Pages](https://github.com/githubocto/flat-demo-covid-dashboard/tree/main/dashboard) to showcase that data.

**Reply by mskyttner** [reply]
Ah! Great, thanks! Just what I was looking for!
